annorxiver_modules/document_helper.py

```diff
@@ -115,6 +136,102 @@ def generate_doc_vector(model, document_path, xpath, filter_tags=filter_tag_list
     return []
 
 
+def generate_doc_vector_parallel(
+    model, xpath, document_queue, vector_queue, filter_tags=filter_tag_list
+):
+    """
+    This method is designed to construct document vectors for a given xml document.
+    Every document has specific tags that are striped in order to have accurate embeddings
```

Comment:
What are these tags? Would it help to have an example in this description?

Reply:
For bioRxiv's xml documents these tags include extraneous text that are not conducive for document parsing (e.g. figures, metadata etc.). I'll put an example.